### DISCUSSION TITLE
Welcome to Wind-Turbine-Control - Simulink Discussions!

### DISCUSSION THREAD

**Discussion by borgestassio** [borgestassio/Wind-Turbine-Control#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheAfzalAmanullah** [comment]
Even after changing in #66 line "ServoData/DISCON_x64.dll" I got the error.

Error using Run_OpenLoop (line 30)
Error reported by S-function 'FAST_SFunc' in 'OpenLoop/FAST Nonlinear Wind Turbine/S-Function':
FAST_InitializeAll:SrvD_Init:ValidatePrimaryData:Yaw angle and rate are not commanded from Simulink model.
ValidatePrimaryData:Pitch angles are not commanded from Simulink model.
ValidatePrimaryData:Generator torque and power are not commanded from Simulink model.
ValidatePrimaryData:HSS brake is not commanded from Simulink model.
SrvD_Init:BladedInterface_Init:The dynamic library ..\..\CertTest\5MW_Baseline/ServoData/DISCON_win32.dll could not be loaded. Check that the file exists in the specified location and that it is compiled for 64-bit applications.

What should I do in this situation?

**Comment by kenilyes** [comment]
hello may you help me
